examples/eks-feature-demo/blueprint.yaml

```diff
@@ -0,0 +1,580 @@
+tosca_definitions_version: cloudify_dsl_1_3
```

Comment:
I'd like to add this example to cloudify-community/blueprint-examples/kubernetes